Modules/Filtering/ImageStatistics/include/itkMinimumProjectionImageFilter.h

```diff
@@ -54,13 +54,38 @@ class MinimumAccumulator
   inline void
   Initialize()
   {
-    m_Minimum = NumericTraits<TInputPixel>::max();
+    if constexpr (std::is_same<TInputPixel, typename NumericTraits<TInputPixel>::ValueType>::value)
+    {
+      m_Minimum = NumericTraits<TInputPixel>::max();
+    }
+    else
+    {
+      m_Minimum = TInputPixel();
```

Comment:
Will this work correctly for constant-sized vectors such as RGB/RGBA?

Reply:
This is a good question. I was focused on SimpleITK VectorImage support... I am searching for the best way to initialize this for all vector-like types... Any ideas?

Reply:
After this, set this to max component-wise via a for loop.

Reply:
```cpp
if (itk::NumericTraits<TInputPixel>::GetLength(m_Minimum) > 0)
{
  for ...
}
```

Reply:
In fact, we don't even need the `if` - just the for loop is enough. If will devolve into a no-op if length of the vector is zero.

Reply:
Yea, I have looked for a portable fill method long enough.

Reply:
Does `Type::Filled()` work for VLVs? If so, it could be used here.

Reply:
It probably works with enough types.